Game/Source/GDKTestGyms/Private/UserExperienceComponent.cpp

```diff
@@ -76,10 +77,13 @@ void UUserExperienceComponent::OnRep_ClientTimeTicks(int64 OldTicks)
 void UUserExperienceComponent::OnClientOwnershipGained()
 {
 	Super::OnClientOwnershipGained();
+
 	FTimerHandle Timer;
-	FTimerDelegate Delegate;
-	Delegate.BindUObject(this, &UUserExperienceComponent::StartRoundtrip);
-	GetWorld()->GetTimerManager().SetTimer(Timer, Delegate, 1.0f, true);
+	GetWorld()->GetTimerManager().SetTimer(Timer, this, &UUserExperienceComponent::StartRoundtrip, 1.0f, true);
+
+	FTimerHandle PositionCheckTimer;
+	GetWorld()->GetTimerManager().SetTimer(PositionCheckTimer, this, &UUserExperienceComponent::CheckPosition, 10.0f, true, 10.0f);
```

Comment:
Is this safe, will this blow up if `UUserExperienceComponent` get gced from under it?

Reply:
Good point - will unbind on destroy